backend/routes/tours.js

```diff
@@ -161,20 +182,42 @@ router.put('/updateTourDateTime', async (request, response, next) => {
 
         if(newDate && newTime){
             const results = await request.models.tour.updateTourDateTime(tour, newDate, newTime);
-            response.status(200).json(results);
+            if(results){
+                const check = await request.models.tour.getToursbyName(tour);
+                response.status(200).json(check);
+            }
+            else{
+                console.error('There was an error in PUT /tours/updateTourDateTime');
+                response.status(500).json();
+            }
         }
         else if(newDate){
             const results = await request.models.tour.updateTourDate(tour, newDate);
-            response.status(200).json(results);
+            if(results){
+                const check = await request.models.tour.getToursbyName(tour);
+                response.status(200).json(check);
+            }
+            else{
+                console.error('There was an error in PUT /tours/updateTourDateTime');
+                response.status(500).json();
+            }
         }
         else if(newTime){
             const results = await request.models.tour.updateTourTime(tour, newTime);
-            response.status(200).json(results);
+            if(results){
+                const check = await request.models.tour.getToursbyName(tour);
+                response.status(200).json(check);
+            }
+            else{
+                console.error('There was an error in PUT /tours/updateTourDateTime');
+                response.status(500).json();
+            }
         }
         else{
             response.status(400).json(); //if there's nothing in the body
```

Comment:
This is fine as is, but you can also put the same message in the .json() part of the response.